sessionctx/variable/sysvar_test.go

```diff
@@ -659,7 +659,7 @@ func TestSettersandGetters(t *testing.T) {
 			case TiDBEnableChangeMultiSchema, TiDBDDLReorgBatchSize,
 				TiDBMaxDeltaSchemaCount, InitConnect, MaxPreparedStmtCount,
 				TiDBDDLReorgWorkerCount, TiDBDDLErrorCountLimit, TiDBRowFormatVersion,
-				TiDBEnableTelemetry, TiDBEnablePointGetCache:
+				TiDBEnableTelemetry, TiDBEnablePointGetCache, TiDBEnableAutoAnalyze:
```

Comment:
I tested this locally. You should be able to revert this change - `TiDBEnableAutoAnalyze` doesn't need to be mentioned here.